setup.cfg

```diff
@@ -118,6 +118,7 @@ install_requires =
     markupsafe>=1.1.1
     marshmallow-oneofschema>=2.0.1
     mdit-py-plugins>=0.3.0
+    mock>=5.0.2;python_version<="3.7"
```

Comment:
We dont support Python 3.7 any longer.
Where did you notice failure related to Python 3.7?

Reply:
In the stale PR https://github.com/apache/airflow/actions/runs/4885483415/jobs/8719703717?pr=31010
I understand Py 3.7 support will be dropped only in the next Airflow release, but I didn't realize the test suites have already dropped it, sorry about that.

I've reverted the Py 3.7 related changes.

Reply:
This is provider code :)
You can see in the change log that we already dropped support
https://airflow.apache.org/docs/apache-airflow-providers-github/stable/index.html#id1

Python 3.7 is relevant only for Airflow 2.6 releases

Reply:
Ah TIL, this is my first time contributing to Airflow, I've only been a user until now.
Thanks for the review 😄